source/js/buyers-guide/search/member-functions.js

```diff
@@ -156,43 +156,42 @@ export function setupGoBackToAll(instance) {
 
 /**
  * Attach event listeners to the Product Review nav links
- *
- * @param {*} instance
- *
- * @todo See if we can make this DRYer by making use of the toggleProductReviewView function?
  */
-export function setupReviewLinks(instance) {
+export function setupReviewLinks() {
```

Comment:
I cleaned up the code in function `setupReviewLinks` and named function `toggleProductReviewView` (now called `showProductReviewView`).

What `setupReviewLinks` does is to ensure clicking the "Product Reviews" link 
- triggers the "product review view", 
- updates the url hash, 
- and collapses the mobile nav screen if it's open.

I cleaned up its implementation so it utilizes `showProductReviewView()` instead of repeating the same code.